src/sass/_theme_variables.sass

```diff
@@ -66,4 +66,4 @@ $custom-font-family:                  "Roboto Slab", "ff-tisa-web-pro", "Georgia
 $custom-font-family2:                 Georgia, serif
 $code-font-family:                    SFMono-Regular, Menlo, Monaco, Consolas, "Liberation Mono", "Courier New", Courier, monospace
 
-$font-display:                        swap
+$font-display:                        block
```

Comment:
I wonder if we might try `fallback` here:


> fallback: Acts as a compromise between the auto and swap values. The browser will hide the text for about 100ms and, if the font has not yet been downloaded, will use the fallback text. It will swap to the new font after it is downloaded, but only during a short swap period (probably 3 seconds).

It seems like it might be a better experience for folks with really slow connections, since the font won't change after they start reading the page. I don't feel strongly tho.

Reply:
`fallback` has it's own issues. You'll still see the unstyled text flash, as the block period is short, and then if the fonts don't load in the swap period, the unstyled text will stick around without ever showing the downloaded fonts -- and the downloaded fonts are still transferred, just never used.